src/styles/page.module.css

```diff
@@ -15,3 +15,10 @@
 	align-items: center;
 	padding-left: var(--size-xs);
 }
+
+.nav {
```

Comment:
Style nitpick that you can feel free to push back on: I prefer not to use abbreviations in variable or classnames (this one is innocuous but some can really make the codebase less readable over time). Can you do `.navigation` instead?